spec/rubocop/cop/rspec/message_chain_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe RuboCop::Cop::RSpec::MessageChain do
+  subject(:cop) { described_class.new }
+
+  it 'finds `receive_message_chain`' do
+    expect_violation(<<-RUBY)
+      before do
+        allow(foo).to receive_message_chain(:one, :two) { :three }
+                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Avoid stubbing using `receive_message_chain`
+      end
+    RUBY
+  end
+
+  it 'finds old `should_receive` syntax' do
+    expect_violation(<<-RUBY)
+      before do
+        foo.stub_chain(:one, :two).and_return(:three)
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^ Avoid stubbing using `stub_chain`
```

Comment:
I can call `add_offense` with `:selector` instead of `:expression` and get highlighting on only `stub_chain` and `receive_message_chain` – without the arguments.